### PR TITLE
Add slashing feeds to slasher

### PR DESCRIPTION
**Pull request by 0xKiwi**
This PR adds a slashing feed for the slasher to send any detected slashings into.

### CODE REVIEW

File: beacon-chain/slasher/detect_blocks.go [hunk 1]
```diff
@@ -28,6 +30,10 @@ func (s *Service) detectSlashableBlocks(
 		}
 		if isDoubleProposal(proposedBlocks[i].SigningRoot, existingProposal.SigningRoot) {
 			doubleProposalsTotal.Inc()
+			s.blockSlashingsChan <- &ethpb.ProposerSlashing{
```
Comment (rauljordan) [inline beacon-chain/slasher/detect_blocks.go:33]: This isn't right, we should instead just send over a feed called proposerSlashingsFeed

File: beacon-chain/slasher/broadcast.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package slasher
```
Comment (rauljordan) [inline beacon-chain/slasher/broadcast.go:1]: We don't need this file

File: beacon-chain/slasher/detect_blocks.go [hunk 3]
```diff
@@ -3,6 +3,8 @@ package slasher
 import (
 	"context"
 
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+
```
Comment (terencechain) [inline beacon-chain/slasher/detect_blocks.go:7]: Remove line

**Review by terencechain — APPROVED** [review]

File: beacon-chain/slasher/detect_blocks.go (hunk 3 above)
Comment (rauljordan) [inline beacon-chain/slasher/detect_blocks.go:7]:
```suggestion
```

**Review by rauljordan — APPROVED** [review]

**Pull request merged**